airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGenerator.java

```diff
@@ -183,12 +183,13 @@ THEN JSON_QUERY(`_airbyte_data`, '$."${column_name}"')
   // the SQLGenerator?
   public static StandardSQLTypeName toDialectType(final AirbyteProtocolType airbyteProtocolType) {
     return switch (airbyteProtocolType) {
-      case STRING, TIME_WITH_TIMEZONE -> StandardSQLTypeName.STRING;
+      case STRING -> StandardSQLTypeName.STRING;
```

Comment:
Didn't intellij keep nagging you to merge the duplicate branches ?  I guess this is more readable.

Reply:
yeah, bad merge on my part